GUI/SearchTaskWindow.xaml

```diff
@@ -222,12 +396,62 @@
                             </GroupBox>
                             <GroupBox Header="Mass Difference Acceptor Criterion">
                                 <StackPanel>
-                                    <RadioButton x:Name="massDiffAcceptExact" Content="Exact" />
-                                    <RadioButton x:Name="massDiffAccept1mm" Content="1 Missed Monoisotopic Peak" />
-                                    <RadioButton x:Name="massDiffAccept2mm" Content="1 or 2 Missed Monoisotopic Peaks" />
-                                    <RadioButton x:Name="massDiffAccept3mm" Content="1, 2, or 3 Missed Monoisotopic Peaks" />
-                                    <RadioButton x:Name="massDiffAccept187" Content="-187 and Up" />
-                                    <RadioButton x:Name="massDiffAcceptOpen" Content="Accept all" />
+                                    <RadioButton x:Name="massDiffAcceptExact" Content="Exact" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <RadioButton.ToolTip>
+                                            <TextBlock>
+                                                    Basic search where the precursor mass must match between the experimental and theoretical.
+                                            </TextBlock>
+                                        </RadioButton.ToolTip>
+                                    </RadioButton>
+                                    <RadioButton x:Name="massDiffAccept1mm" Content="1 Missed Monoisotopic Peak" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <RadioButton.ToolTip>
+                                            <TextBlock>
+                                                The reported precursor mass is not always corrected, because of missed monoisotopic errors.
```

Comment:
Basic search where the precursor mass and experimental mass may be different by 1 Da to allow for a 1 Da monoisotopic mass error.

Reply:
duplicate below for 2mm and 3mm